servers/express.js

```diff
@@ -118,6 +207,8 @@ server.start()
     // eslint-disable-next-line no-console
     console.log(process.pid);
   })
-  .then(async () => {
-    await collector();
+  .then(async() => {
+    if (pathToLogFile) {
+      await collector();
```

Comment:
Since the promises in JavaScript are monads you don't need to await here, just return the promise.

Reply:
it doesn't need to be async either, so simply:

```
.then(() => pathToLogFile && collector())
```

or, more explicitly
```
.then(() => {
  if (pathToLogFile) {
    return collector();
  }
  return null;
)
```